src/java.base/share/classes/java/lang/VirtualThread.java

```diff
@@ -96,37 +96,37 @@ final class VirtualThread extends BaseVirtualThread {
      *  RUNNING -> PARKING         // Thread parking with LockSupport.park
      *  PARKING -> PARKED          // cont.yield successful, parked indefinitely
      *  PARKING -> PINNED          // cont.yield failed, parked indefinitely on carrier
-     *   PARKED -> RUNNABLE        // unparked, schedule to continue
+     *   PARKED -> UNPARKED        // unparked, may be scheduled to continue
      *   PINNED -> RUNNING         // unparked, continue execution on same carrier
+     * UNPARKED -> RUNNING         // continue execution after park
      *
      *       RUNNING -> TIMED_PARKING   // Thread parking with LockSupport.parkNanos
      * TIMED_PARKING -> TIMED_PARKED    // cont.yield successful, timed-parked
      * TIMED_PARKING -> TIMED_PINNED    // cont.yield failed, timed-parked on carrier
-     *  TIMED_PARKED -> RUNNABLE        // unparked, schedule to continue
+     *  TIMED_PARKED -> UNPARKED        // unparked, may be scheduled to continue
      *  TIMED_PINNED -> RUNNING         // unparked, continue execution on same carrier
      *
-     * RUNNABLE -> RUNNING         // continue execution
-     *
      *  RUNNING -> YIELDING        // Thread.yield
-     * YIELDING -> RUNNABLE        // yield successful
-     * YIELDING -> RUNNING         // yield failed
+     * YIELDING -> YIELDED         // cont.yield successful, may be scheduled to continue
+     * YIELDING -> RUNNING         // cont.yield failed
+     *  YIELDED -> RUNNING         // continue execution after Thread.yield
      */
     private static final int NEW      = 0;
     private static final int STARTED  = 1;
-    private static final int RUNNABLE = 2;     // runnable-unmounted
-    private static final int RUNNING  = 3;     // runnable-mounted
-
-    // untimed parking
-    private static final int PARKING  = 4;
-    private static final int PARKED   = 5;     // unmounted
-    private static final int PINNED   = 6;     // mounted
+    private static final int RUNNING  = 2;     // runnable-unmounted
```

Comment:
Just want to make sure this change is intentional.
Before the comment was:  `// runnable-mounted`.

Reply:
> Just want to make sure this change is intentional. Before the comment was: `// runnable-mounted`.

Well spotted, the wrong comment line was moved.